snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksum.java

```diff
@@ -62,7 +62,7 @@ private static boolean isNotMetadataFile(final Path file) {
 
   public static void persist(final Path checksumPath, final ImmutableChecksumsSFV checksum)
       throws IOException {
-    try(final var stream = new FileOutputStream(checksumPath.toFile())) {
+    try (final var stream = new FileOutputStream(checksumPath.toFile())) {
```

Comment:
ups :see_no_evil:

Reply:
Yeah :D seems to be from the other PR :D